product_supplierinfo_import_margin/wizards/product_supplierinfo_import.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class ProductSupplierInfoImport(models.TransientModel):
+    _inherit = "product.supplierinfo.import"
+
+    sale_margin = fields.Float(required=True)
```

Comment:
The required attribute on float fields has no effect, so it can be removed.
```suggestion
    sale_margin = fields.Float()
```